framework/src/base/ScalarCoupleable.C

```diff
@@ -160,17 +171,38 @@ ScalarCoupleable::coupledScalarValueOlder(const std::string & var_name, unsigned
 VariableValue &
 ScalarCoupleable::coupledScalarDot(const std::string & var_name, unsigned int comp)
 {
+  checkVar(var_name, comp);
   MooseVariableScalar * var = getScalarVar(var_name, comp);
   return var->uDot();
 }
 
 VariableValue &
 ScalarCoupleable::coupledScalarDotDu(const std::string & var_name, unsigned int comp)
 {
+  checkVar(var_name, comp);
   MooseVariableScalar * var = getScalarVar(var_name, comp);
   return var->duDotDu();
 }
 
+void
+ScalarCoupleable::checkVar(const std::string & var_name, unsigned int comp)
+{
```

Comment:
I'm getting an unused parameter warning for `comp` here. I can't see that you are using it anywhere?

Reply:
Right, I got that in debug mode as well and meant to either comment it out or remove it. Apparently I forgot to do so. I will fix that... Thanks!